linters/elixir/.credo.exs

```diff
@@ -0,0 +1,21 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "src/", "web/", "apps/"],
+        excluded: [~r"/_build/", ~r"/deps/"]
+      },
+      requires: [],
+      check_for_updates: true,
+      strict: true,
+      color: true,
+
+      checks: [
+        {Credo.Check.Design.TagTODO, exit_status: 0},
+        {Credo.Check.Readability.ModuleDoc, false},
+        {Credo.Check.Readability.Specs, false},
```

Comment:
Disables the enforce of a typespec (`@spec`) when overriding behaviours. An example would be to create a mix app with a module that contains `use Application`. The required `start` function, without disabling that check, needs to have a `@spec` specification.

It seemed odd to me, given the name of the check, but it doesn't seem to enforce the presence of a typespec in any other function. I couldn't find documentation for that as well.